src/middleware.ts

```diff
@@ -151,10 +151,12 @@ export const config = {
 		 * - /api/ (API routes)
 		 * - static (static files)
 		 * - _next/static (Next.js static files)
+		 * - _next/image (Next.js optimized images)
+		 * - _next/data (Next.js static data)
 		 * - img (image assets)
 		 * - favicon.ico (favicon file)
 		 * - icon (hashicorp logo)
 		 */
-		'/((?!api\\/|static|_next\\/static|img|favicon.ico|icon).*)',
+		'/((?!api\\/|static|_next\\/(static|data|image)|img|favicon.ico|icon).*)',
```

Comment:
How bout:
```suggestion
		'/((?!api\\/|static|_next\\/|img|favicon.ico|icon).*)',
```

I don't think there's any reason to run the middleware on any of the `/_next/` requests.